tools/ctl/ide/ui.go

```diff
@@ -416,7 +424,7 @@ func (e *explorer) LoadTree() error {
 	}
 	byBench := make(map[string][]string)
 	for _, run := range runs {
-		if run.Type == benchmark.AttestMode {
+		if !e.settings.includeAttest && run.Type == benchmark.AttestMode {
```

Comment:
do we even need this constraint? would an alternative be some indication in the UI?

Reply:
I think eventually we will want some setting/filters on the explorer with a way to adjust them in the UI (I'm thinking like filter based on error message, or grouping based on log regex) but I can leave the settings construct out of this PR and just add an indication. The indication is useful regardless.